_examples/ceph-source-example.adoc

```diff
@@ -60,15 +73,19 @@ No architecture, this is a connectivity example.
 
 2. Start a Jupyter notebook with the commands:
 
-   oc new-app rimolive/notebook
-   oc expose svc/notebook
+  oc new-app radanalyticsio/base-notebook \
+    -e JUPYTER_NOTEBOOK_PASSWORD=supersecret \
+    -e JUPYTER_NOTEBOOK_X_INCLUDE=https://radanalytics.io/assets/s3-source-example/s3-source-example.ipynb
+  oc expose svc/base-notebook
 
-3. Find the connection url using the following command:
+3. After the notebook pod shows `Ready` status, find in the logs the connection URL with the command:
```

Comment:
this step shouldn't be needed as we specified a password above, we should just tell the users to click on the link for the notebook and use the password